_schemaV2.graphql

```diff
@@ -18910,6 +18979,8 @@ enum UserInterestInterestType {
   GENE
 }
 
+union UserInterestOrError = CreateUserInterestFailure | UserInterest
```

Comment:
The mutation returns an array of this union type: either `CreateUserInterestFailure` or `UserInterest`.

```
  "userInterestsOrErrors": [
      {
        "category": "COLLECTED_BEFORE",
        "interest": {
          "name": "Artist Name",
        },
      },
     {
       "mutationError": {
         "type": "error",
         "message": "Interest Not Found"
        }
      },
    ],
```